syntax = "proto3";

package instruction;

service GRPCService {
    rpc GetInstruction (InstructionRequest) returns (InstructionResponse) {}
}

message InstructionRequest {
    bytes frame1 = 1;
    bytes frame2 = 2;
    bytes frame3 = 3;
    bytes frame4 = 4;
    string type_instruction = 5;
}

message InstructionResponse {
    string instruction_str = 1;
}

// 